Microsoft.DotNet.Interactive.Tests/LanguageServiceTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.LanguageService;
+using Microsoft.DotNet.Interactive.Tests.Utility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Interactive.Tests
+{
+    public class LanguageServiceTests : LanguageKernelTestBase
+    {
+        public LanguageServiceTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private Task<IKernelCommandResult> SendHoverRequest(KernelBase kernel, string code, int line, int character)
+        {
+            var command = new RequestHoverTextCommand(RequestHoverTextCommand.MakeDataUriFromContents(code), new Position(line, character));
+            return kernel.SendAsync(command);
+        }
+
+        [Theory]
+        [InlineData(Language.FSharp)]
+        [InlineData(Language.PowerShell)]
+        public async Task HoverOnUnsupportedLanguageServiceReturnsNothing(Language language)
```

Comment:
can we align the naming pattern to the other tests like

```lsp_textDocument_hover_returns_expected_result```

Reply:
Will do.